src/cli/help.rs

```diff
@@ -1,5 +1,8 @@
+use anstyle::Style;
 use clap_cargo::style::{HEADER, LITERAL, PLACEHOLDER};
 
+const SUBHEADER: Style = Style::new().bold();
```

Comment:
Nit: sorry, one more that I forgot in the last round: please put this at the bottom.